static/docs/commands-reference/run.md

```diff
@@ -125,12 +125,11 @@ pipeline.
   for confirmation.
 
 - `--ignore-build-cache` - if an exactly equal DVC-file exists (same list of
-  outputs and inputs, the same command to run) which has been already executed,
-  and is up to date, with option `dvc run` won't execute the command again by
-  default (thus "build cache"). This option gives a way to forcefully run the
-  command anyway. It's useful if the command is considered non-deterministic for
-  some reason (meaning it produces different outputs from the same list of
-  inputs).
+  outputs and inputs, the same command to run which has been already executed),
```

Comment:
I think it's import that ) is after run here

Reply:
Good catch, thanks. Fixed in 46aa961